### PR TITLE
Add Eventer to e2e test yaml

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: deploy/kube-config/standalone-test/heapster-controller.yaml [hunk 1]
```diff
@@ -22,10 +22,22 @@ spec:
         command:
         - /heapster
         - --source=kubernetes:https://kubernetes.default
+        - --sink=log
```
Comment (piosz) [inline deploy/kube-config/standalone-test/heapster-controller.yaml]: I'm afraid there is too many logs that come from log sink.

**Comment by piosz** [comment]
LGTM

File: deploy/kube-config/standalone-test/heapster-controller.yaml (hunk 1 above)
Comment (mwielgus) [inline deploy/kube-config/standalone-test/heapster-controller.yaml]: This is only for tests.

**Comment by mwielgus** [comment]
Some strange jenkins flakiness - ignoring for now as it seems to be unrelated to the pr.

**Pull request merged**